sdk/reserved-account-keys/Cargo.toml

```diff
@@ -20,6 +20,7 @@ solana-frozen-abi-macro = { workspace = true, optional = true, features = [
 ] }
 solana-pubkey = { workspace = true, default-features = false }
 solana-sdk-ids = { workspace = true }
+solana-secp256r1-program = { workspace = true }
```

Comment:
I would have asked you to put the secp256r1 program ID in solana-sdk-ids so solana-reserved-account-keys wouldn't end up with a solana-secp256r1-program dependency

Reply:
Will coordinate with sam to make sure https://github.com/anza-xyz/agave/pull/3663 is backported together with this 👍

Reply:
Looks like we don't have to backport since the sdk-ids split isn't in 2.1. Will just get this merged into master 🫡